art/detection/subsetscanning/detector.py

```diff
@@ -0,0 +1,184 @@
+""" Fast generalized subset scan based detector"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+import logging
+import six
+
+from sklearn import metrics
+import numpy as np
+
+from art.classifiers import Classifier
+from art.detection.subsetscanning.scanner import Scanner
+
+logger = logging.getLogger(__name__)
+
+
+class SubsetScanningDetector(Classifier):
+    """ Fast generalized subset scan based detector"""
```

Comment:
Could you please add documentation of the arguments to __init__ to the class docstring similar to the other classes of ART?